monai/handlers/mlflow_handler.py

```diff
@@ -72,11 +76,21 @@ class MLFlowHandler:
         state_attributes: expected attributes from `engine.state`, if provided, will extract them
             when epoch completed.
         tag_name: when iteration output is a scalar, `tag_name` is used to track, defaults to `'Loss'`.
+        experiment_name: name for an experiment, defaults to `default_experiment`.
+        run_name: name for run in an experiment.
+        experiment_param: a dict recording parameters which will not change through whole experiment,
+            like torch version, cuda version and so on.
+        artifacts: paths to images that need to be recorded after a whole run.
+        optimizer_param_names: parameters' name in optimizer that need to be record during runing,
+            defaults to `["lr"]`.
 
     For more details of MLFlow usage, please refer to: https://mlflow.org/docs/latest/index.html.
 
     """
 
+    default_log_param_list = ["network", "device", "optimizer", "loss_function"]
+    default_attr_name = ["seed", "max_epochs", "epoch_length"]
```

Comment:
The `engine.state.seed` is deprecated in ignite and MONAI, I think no need to track it.
CC @vfdev-5 to help confirm it.

Thanks in advance.